app/javascript/dashboard/helper/snoozeHelpers.js

```diff
@@ -0,0 +1,13 @@
+import { getUnixTime, format } from 'date-fns';
+import * as chrono from 'chrono-node';
+
+export const findSnoozeTime = snoozeType => {
```

Comment:
I noticed we don't let users type arbritrary strings and limit them to the options from `SNOOZE_CONVERSATION_ACTIONS`, in that case we may not need `chrono` library, instead we can just define the method ourselves for each of the available options

Reply:
@scmmishra 
- Regarding the alignment of the title and close button, I am using the `woot-modal-header` and some modifications are necessary. It would be best to treat this as a separate task since it is utilized on multiple screens.
- Calendar style issues. Do you mean something like this?

![CleanShot 2023-06-14 at 15 26 40](https://github.com/chatwoot/chatwoot/assets/12408980/c00b89e9-6464-4497-9552-b559042a3bb6)

Reply:
@scmmishra @pranavrajs @nithindavid Any feedback on the custom snooze modal?

![CleanShot 2023-06-14 at 22 08 40](https://github.com/chatwoot/chatwoot/assets/12408980/1cdf2f3b-7d6b-42d0-b316-426cc952a758)